src/graphics/webgpu/webgpu-render-target.js

```diff
@@ -0,0 +1,202 @@
+import { Debug } from '../../core/debug.js';
+
+/** @typedef {import('../render-pass.js').RenderPass} RenderPass */
+/** @typedef {import('../render-target.js').RenderTarget} RenderTarget */
+/** @typedef {import('../webgpu/webgpu-graphics-device.js').WebgpuGraphicsDevice} WebgpuGraphicsDevice */
+
+/**
+ * A WebGPU implementation of the RenderTarget.
+ *
+ * @ignore
+ */
+class WebgpuRenderTarget {
+    /** @type {boolean} */
```

Comment:
is this needed? doesn't the type system infer boolean from the initial value?

Reply:
It would be needed for JSDoc, but since the class is under `@ignore`, the type doesn't need to be specified twice for the sake of Intellisense at least.

Reply:
I specify types for my readability when declaring class.